timesheet_automatic_remind/models/timesheet_employee_remind.py

```diff
@@ -0,0 +1,291 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import datetime
+import time
+import pytz
+from openerp import fields, models, api
+
+
+class TimesheetReport(models.TransientModel):
```

Comment:
General comments:

1. Add documentation for all the functions. It's not easy to guess the purpose of the function. And it's almost impossible to guess the purpose of each parameter
2. A lot of parameters have a default  value. Based on my understanding of the code, it doesn't make sense. Please specify in the documentation what values can take the parameters and make sure only the parameters that are really **optional** have a default value